config/set/nette-30-composer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Composer\Rector\ChangePackageVersionRector;
+use Rector\Composer\Rector\RemovePackageRector;
+use Rector\Composer\Rector\ReplacePackageAndVersionRector;
+use Rector\Composer\ValueObject\PackageAndVersion;
+use Rector\Composer\ValueObject\ReplacePackageAndVersion;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symplify\SymfonyPhpConfig\ValueObjectInliner;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+
+    $services->set(ChangePackageVersionRector::class)
+        ->call('configure', [[
+            ChangePackageVersionRector::PACKAGES_AND_VERSIONS => ValueObjectInliner::inline([
```

Comment:
Did you test it with multiple sets? For example nette30 + php7.1 (which change php to ^7.1)? Did not work for me in old code.

Reply:
Not sure what you mean. Test case would help

Reply:
The best practise is to use one set at a time during big upgrade.
Only then add multiple sets to CI.

Reply:
The best practise is to use one set at a time during big upgrade.
Only then add multiple sets to CI.